code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -76,6 +83,12 @@
 		user.drop_item()
 		qdel(O)
 		return
+	else if (istype(O, /obj/item/slimepotion/mutator))
+		mutators++
+		to_chat(user, "<span class='notice'>You feed [O] to [src]. It now has [mutators] slime mutation potions stored.</span>")
+		user.drop_item()
```

Comment:
this should be checked for first, otherwise, you're going to have issue with `NO_DROP`.